src/components/Footer/Footer.js

```diff
@@ -2,10 +2,11 @@ import React from 'react';
 
 import './Footer.css';
 
-function Footer() {
+function Footer(props) {
     return (
-        <footer className='footer'>
-            <p>Written and coded by <a href="https://www.freecodecamp.org/d-pagey" 
+        <footer className={props.footerClass}>
```

Comment:
maybe clearer, if Footer accepts "className" prop, then
`<footer className={props.className}>`

Reply:
in parent level, `<Footer className={state.footerClass}>`